docs/design_decisions.md

```diff
@@ -1,5 +1,9 @@
 # Design decisions HaalCentraal API's
-Dit document beschrijft ontwerpkeuzes die gemaakt zijn voor het ontwerpen en specificeren van de API's binnen het programma HaalCentraal
+In de onderliggende Design Decisions zijn de inzichten vastgelegd die zijn opgedaan bij het ontwerpen en specificeren van API's binnen het programma HaalCentraal. Ze geven steeds de huidige status aan van een leerproces en het is dus onderhevig aan voortschrijdend inzicht. Om die reden kan het zijn dat de in het HaalCentraal programma vervaardigde specificaties niet voldoen aan alle Design Decisions.
```

Comment:
Om die reden kan het zijn dat de specificaties die vroeg in het Haal Centraal programma vervaardigd zijn niet voldoen aan alle Design Decisions.